src/core/selection.coffee

```diff
@@ -122,6 +122,9 @@ class Selection
     return this._decodePosition(leaf.node, offset)
 
   _positionToIndex: (node, offset) ->
+    if dom.isIE(10) and node.innerHTML == '<br>' and offset == 1
```

Comment:
Can you collapse this into one line:

offset = 0 if dom.isIE(10) and node.tagName == 'BR' and offset == 1